<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QDialog" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="genderLabel">
          <property name="text">
           <string>Gender:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="birthYearLabel">
          <property name="text">
           <string>Birth year:  </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="genderBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="birthYearBox"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="isDeadCheckBox">
        <property name="text">
         <string>Person is dead</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="deathYearLabel">
          <property name="text">
           <string>Death year:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="deathYearBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addPersonButton">
        <property name="text">
         <string>Add person</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>cancelButton</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>nameLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
